docs/4_the-formal-stuff/career-development.md

```diff
@@ -123,6 +123,50 @@ Our engineering ladders have the following 5 axes (categories):
 
 ---
 
+## RnD Levels
```

Comment:
Is it better if we organize this into subpages like `Typescript -> Monorepo` is?

Reply:
They have both confirmed that they don't need so only Research now